frontend/plugins/table-plugin/config.ts

```diff
@@ -11,14 +12,16 @@ import {
     toggleHeaderColumn,
     toggleHeaderRow,
 } from "prosemirror-tables";
-import { translate } from "@/i18n/translations";
-import {
-    isHeaderRowActive,
-    isHeaderColumnActive,
-    ButtonOrDropdown,
-} from "@/components/table-toolbar";
+import { isHeaderColumnActive, isHeaderRowActive } from "@/utils";
+import { CreateMenuItems } from "../toolbar-plugin";
+// import { ImageDOMAttrs } from "@/schema/nodes/image";
```

Comment:
so here you're commenting out the import, but then import ImageDOMAttrs again, and i am still not sure what the image functionality needs to do in a Table.